embed/extmod/modtrezorcrypto/modtrezorcrypto-ed25519.h

```diff
@@ -91,6 +91,33 @@ STATIC mp_obj_t mod_trezorcrypto_ed25519_sign(size_t n_args, const mp_obj_t *arg
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(mod_trezorcrypto_ed25519_sign_obj, 2, 3, mod_trezorcrypto_ed25519_sign);
 
+/// def sign_cardano(secret_key: bytes, secret_extenion: bytes, message: bytes) -> bytes:
+///     '''
+///     Uses secret key to produce the cardano signature of message.
+///     '''
+STATIC mp_obj_t mod_trezorcrypto_ed25519_sign_cardano(mp_obj_t secret_key, mp_obj_t secret_extenion, mp_obj_t message) {
```

Comment:
Rename to `mod_trezorcrypto_ed25519_sign_ext`, also there are couple of typos `secret_extenion` -> `secret_extension`.